### PR TITLE
Added a fake label PNG to the label preview

### LINKED ISSUE #507: [Labels] Preview step

### PR DESCRIPTION
**Pull request by DanReyLop**
Fixes #507 (for now).

This PR adds a fake label to the "Preview" sidebar on the Labels purchase UI. That fake (static) label will only be shown if all the steps on the form are correct.

Screenshot:
preview

@allendav @jeffstieler @allendav @robobot3000

### CODE REVIEW

File: classes/class-wc-connect-shipping-label.php [hunk 1]
```diff
@@ -238,6 +238,7 @@ public function meta_box( $post ) {
 				'purchaseURL'             => get_rest_url( null, '/wc/v1/connect/shipping-label' ),
 				'addressNormalizationURL' => get_rest_url( null, '/wc/v1/connect/normalize-address' ),
 				'getRatesURL'             => get_rest_url( null, '/wc/v1/connect/shipping-rates' ),
+				'labelPreviewURL'         => plugins_url( 'assets/fake_label.png', dirname( __FILE__ ) ),
```
Comment (allendav) [inline classes/class-wc-connect-shipping-label.php]: We will probably need to have labels for other carriers soon, so let's use the carrier name in the filename to make this easier to distinguish between other carrier's labels, e.g. `usps_sample_label.png`

**Comment by allendav** [comment]
Works great. I love it. One tweak though - please don't lose the gray container and do center the preview label in the container

screen shot 2016-09-12 at 1 10 07 pm

**Comment by allendav** [comment]
@DanReyLop - also, please photoshop out the following in the sample label png:

* "endicia.com" in the upper right
* "Fake Device ID" in the upper right
* "PARCEL SELECT GROUND TM" above the origin address - just keep the "USPS" there and drag it to the center
*

**Comment by DanReyLop** [comment]
@allendav Done, and done. Ready for the second pass :)

**Comment by allendav** [comment]
Ty! :shipit:

**Pull request merged**